var/spack/repos/builtin/packages/tensorflow-estimator/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class TensorflowEstimator(Package):
+    """TensorFlow Estimator is a high-level TensorFlow API that greatly
+    simplifies machine learning programming."""
+
+    homepage = "https://github.com/tensorflow/estimator"
+    url      = "https://github.com/tensorflow/estimator/archive/v1.13.0.tar.gz"
+
+    version('2.0.0-alpha', '739923c39443ec614f5348d7ff7f1d03')
```

Comment:
Looks like 2.0.0 and 2.0.1 are released now, no need to add an alphas version. Also need to convert to sha256.